### PR TITLE
Fix: hard-coded per page

### PR DESCRIPTION
**Pull request by ivke995**

### CODE REVIEW

File: apps/stack/src/extract/extract-namespace-members.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ const extractNamespaceMembersPage = async ({ namespace, repositoryId, sourceCont
     externalNamespaceId: namespace.externalId,
     namespaceName: namespace.name,
     repositoryId,
-    perPage: perPage,
+    perPage: perPage || Number(Config.PER_PAGE),
```
Comment (Ante-Koceic) [inline apps/stack/src/extract/extract-namespace-members.ts:93]: Is there a reason for

perPage: perPage || Number(Config.PER_PAGE),

and not just 

perPage: Number(Config.PER_PAGE)

like in the extract-merge-requests.ts file?

Comment (ivke995) [reply apps/stack/src/extract/extract-namespace-members.ts]: With new commit refactored to be consistent

**Review by davidabram — APPROVED** [review]
:crocodile: :croissant:

**Pull request merged**